### PR TITLE
[FSSDK-9486] chore: Prepare for releasing 3.11.3

### PR DESCRIPTION
**Pull request by mikechu-optimizely**
## Summary
- Increment semantic version number in assemblies
- Update CHANGELOG

## Test plan
- Only release prep. No code paths changed.
- All existing tests should continue to pass.

## Issues
- FSSDK-9472
- FSSDK-9486

### CODE REVIEW

**Comment by mikechu-optimizely** [comment]
PRs #356 & #357 need to be merged into the release-branch before this one.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Optimizely C# SDK Changelog
 
+## 3.11.3
+July 3, 2023
```
Comment (jaeopt) [inline CHANGELOG.md:4]: This date should be changed?

**Review by jaeopt — APPROVED** [review]
LGTM - date in changelog should be updated

**Comment by muzahidul-opti** [comment]
Curious to know why fullstack check is failing.

**Comment by mikechu-optimizely** [comment]
> LGTM - Curious to know why fullstack check is failing.

Great catch @muzahidul-opti I missed that because apparently I don't have the checks configured correctly.  Let me look into both before continuing....

**Comment by mikechu-optimizely** [comment]
Oh wait, branch checks are configured for `master` and we're merging into `release-3.11.3`. 

...but yeah why is optimizely/fullstack-sdk-compatibility-suite  showing failure....hmmm

**Comment by mikechu-optimizely** [comment]
Found it. 

I'm building SDK version 3.11.3, which does not have any coding for ATS/ODP.  FSC is using end-to-end tests (csharp-testapp) that has ODP in play and expects the SDK to have the ODP classes so the [build is failing](https://github.com/optimizely/fullstack-sdk-compatibility-suite/actions/runs/5592201419/jobs/10224335237#step:9:317)

I need to think about how to use an older tag of the csharp-testapp that did not expect ODP to be present 🤔 

@muzahidul-opti

**Comment by mikechu-optimizely** [comment]
Grrr.  Ok. TestApp built successfully using an older non-ODP version, but now FSC itself needs to be run using a version before adding ODP.

[58 failing tests](https://github.com/optimizely/fullstack-sdk-compatibility-suite/actions/runs/5599952561/jobs/10241630981#step:9:11900) all appear to be ODP. 

@muzahidul-opti @jaeopt

**Comment by mikechu-optimizely** [comment]
@yasirfolio3 Is there a way to [run FSC at a point before ODP tests](https://github.com/optimizely/fullstack-sdk-compatibility-suite/tree/b45d804ce7670090341fcbae91f5daa39b03aa94)?

**Comment by mikechu-optimizely** [comment]
Thanks 
@yasirfolio3 for pointing me in the direction of passing e2e testing for non ODP C# 
@muzahidul-opti for catching my oversight on e2e
@jaeopt for your 3 reviews to get this out the door.

**Pull request merged**